.agents/skills/sym-address-comments/scripts/fetch_comments.py

```
@@ -0,0 +1,237 @@
+
+    while True:
+        payload = gh_api_graphql(
+            owner=owner,
+            repo=repo,
+            number=number,
+            comments_cursor=comments_cursor,
+            reviews_cursor=reviews_cursor,
+            threads_cursor=threads_cursor,
+        )
+
+        if "errors" in payload and payload["errors"]:
```

Ruff (0.15.9)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)